### ISSUE TITLE
CVE-2005-3660

### ISSUE THREAD

**Issue by nluedtke** [nluedtke/linux_kernel_cves#7]
Quick research yielded no known fix for this issue. It is unclear if this CVE was fixed via patch or if it was left to be handled by other mitigations.

**Comment by avelardi** [comment]
It looks like this bug was unofficially marked as won't fix. While it may be vendor specific, the Debian Security Advisory noted it was a design limitation:

> Design limitation, for rare corner cases, where this poses a problem advanced
resource management systems can be deployed

Link here: https://security-tracker.debian.org/tracker/CVE-2005-3660

**Comment by nluedtke** [comment]
Yeah, Debian's kernel security repo has it under ignored. with the comment:

> dannf> The fix suggested by idefense includes adding a struct user reference
 dannf> to struct file.  No such thing has gone upstream yet, however.

There are few of these cases where for the earlier vulnerabilities, they were essentially marked as "won't fix" or "working as intended". I am not sure how we want to mark these for the upstream kernel yet.
